WordPress/Classes/ViewRelated/Post/PostEditor+Publish.swift

```diff
@@ -399,32 +377,30 @@ extension PostEditorUtil {
         // a short delay
 
         managedObjectContext.performAndWait {
-            context.post = self.post.createRevision()
+            post = self.post.createRevision()
             ContextManager.sharedInstance().save(managedObjectContext)
         }
     }
 }
 
-extension PostEditorUtil {
 
-    struct Constants {
-        static let autoSavingDelay = Double(0.5)
-    }
+struct PostEditorDebouncerConstants {
+    static let autoSavingDelay = Double(0.5)
+}
 
-    struct Analytics {
-        static let headerStyleValues = ["none", "h1", "h2", "h3", "h4", "h5", "h6"]
-    }
+private struct Analytics {
+    static let headerStyleValues = ["none", "h1", "h2", "h3", "h4", "h5", "h6"]
```

Comment:
It looks like this is not being used.
If that's the case, can we remove it?

Reply:
right! removed 👍